src/main/battlecode/world/RobotControllerImpl.java

```diff
@@ -812,15 +812,15 @@ public boolean canChop(MapLocation loc) {
         boolean correctType = (getType() == RobotType.LUMBERJACK);
         boolean canInteract = canInteractWithTree(loc);
 
-        return correctType && canInteract;
+        return correctType && canInteract && !hasAttacked();
```

Comment:
A slightly better way of doing this might be to implement a paramater-less canChop() method, analogous to the parameter-less canShake() and canWater() methods.